src/main/java/pl/put/poznan/transformer/rest/ScenarioLinesWithoutActorInFirstWordController.java

```diff
@@ -0,0 +1,51 @@
+package pl.put.poznan.transformer.rest;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import pl.put.poznan.transformer.logic.domain.dto.ScenarioDTO;
+import pl.put.poznan.transformer.logic.service.ScenarioLinesWithoutActorInFirstWordService;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+@RestController
+public class ScenarioLinesWithoutActorInFirstWordController {
+    private final ScenarioLinesWithoutActorInFirstWordService scenarioLinesWithoutActorInFirstWordService;
+
+    public ScenarioLinesWithoutActorInFirstWordController(ScenarioLinesWithoutActorInFirstWordService scenarioLinesWithoutActorInFirstWordService) {
+        this.scenarioLinesWithoutActorInFirstWordService = scenarioLinesWithoutActorInFirstWordService;
+    }
+
+    /**
+     * Method: GET<br>
+     * Endpoint: /api/scenario/steps/count<br>
+     * @param scenarioDTO Object representing scenario created from parsed JSON body
+     * @return Object representing below JSON with example values:
+     *      <pre>
+     *      {
+     *          "lines": [
+     *              "0" :"SOMETHING ACTOR",
+     *              "1": "SOMETHING"
+     *              ]
+     *      }
+     *      </pre>
+     */
+    @PostMapping("/scenario/lines/incorrect")
```

Comment:
Wrong http method and name of an endpoint. @ClouddJR @CiesielskiB How to name it? Something like /scenario/steps/not-starting-with-actor? Or maybe /scenario/steps?fiter= and here goes the filter name. That last one would probably need to return all steps when filter is not given.